packages/11ty/_plugins/figures/annotation-factory.js

```diff
@@ -0,0 +1,58 @@
+const chalkFactory = require('~lib/chalk')
+const path = require('path')
+const titleCase = require('~plugins/filters/titleCase')
+const { error } = chalkFactory('Figure Processing:IIIF:Annotations')
+
+/**
+ * Quire Annotation
+ */
```

Comment:
Document here how a Quire Annotation (the Object returned by the factory) corresponds to a Annotation per the IIIF/W3C Annotation spec.